packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/materials/pbrMaterialPropertyGridComponent.tsx

```diff
@@ -1255,6 +1255,13 @@ export class PBRMaterialPropertyGridComponent extends React.Component<IPBRMateri
                         propertyName="useHorizonOcclusion"
                         onPropertyChangedObservable={this.props.onPropertyChangedObservable}
                     />
+                    <CheckBoxLineComponent

```

Comment:
Triggering this doesn't change the shader right now. I think I'm missing something to make this property change force the shader to recompile.

Reply:
Just replace:
```typescript
                    if (this._useAlternateRoughRadiance) {
                        defines.ALTERNATE_ROUGH_RADIANCE = this._useAlternateRoughRadiance;
                    }
```
with
```typescript
                    defines.ALTERNATE_ROUGH_RADIANCE = this._useAlternateRoughRadiance;

```
in `pbrBaseMaterial.ts`.

Reply:
Is this ok now @MiiBond ?